docker-hub/builds/manage-builds.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Manage autobuilds
+description: How to manage autobuilds in Docker Hub
+keywords: autobuilds, automated, docker hub, registry
+---
+> **Note**
+>
+> Automated builds requires a
+> [Docker Pro, Team, or Business subscription](../../subscription/index.md).
+
+## Cancel or retry a build
+
+While a build is in queue or running, a **Cancel** icon appears next to its build
+report link on the **General** tab and on the **Builds** tab. You can also select
+**Cancel** on the build report page, or from the **Timeline** tab's logs
```

Comment:
The page name is formatted as a UI control in the troubleshoot topic. I tried looking for this page in the UI, but I'm unsure how to get to it to verify.

```suggestion
**Cancel** on the **Build report** page, or from the **Timeline** tab's logs
```